src/framework/annotator_mgr.cpp

```diff
@@ -152,12 +155,21 @@ namespace uima {
       assert(iv_vecEntries.empty());
 
       AnnotatorContext & rANC = iv_pEngine->getAnnotatorContext();
-
       AnalysisEngineDescription const & crTAESpecifier = rANC.getTaeSpecifier(); // this method must be added
-
+      const AnalysisEngineMetaData* pEngineMetadata = crTAESpecifier.getAnalysisEngineMetaData();
       assert( ! crTAESpecifier.isPrimitive() );
-      //BSIvector < icu::UnicodeString > const & crVecEngineNames = crTAESpecifier.getAnalysisEngineMetaData()->getFixedFlow()->getNodes();
-      vector < icu::UnicodeString > const & crVecEngineNames = crTAESpecifier.getAnalysisEngineMetaData()->getFlowConstraints()->getNodes();
+
+      // FIXME: This shouldn't have been necessary since FlowContrainst::getFlowContraintsType
```

Comment:
which part is unnecessary?

Reply:
I meant the `FlowContrainst::getFlowContraintsType` method should have been const because it doesn't change the object, so that we shouldn't have to do a CONST_CAST, but that's just a mild annoyance I have